sdk/storage/Azure.ResourceManager.Storage/CHANGELOG.md

```diff
@@ -10,6 +10,18 @@
 
 ### Other Changes
 
+## 1.0.1 (2022-09-23)
```

Comment:
@gaoyp830 For breaking change, you need to bump the major version, e.g. `2.0.0`.